src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  margin: auto;
+  padding: 50px $hor-padding $hor-padding;
```

Comment:
it's not obvious to use variables like this, because it's not unified. 
Why we have $hor-padding and don't have vertical one? 
I would use paddings here without variables